notebooks/257-llava-multimodal-chatbot/257-llava-multimodal-chatbot.ipynb

```diff
@@ -0,0 +1,1222 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #93.        if not image_encoder_path.exists():
</pre><p>Should we saparete logical blocks by blank lines? It will make function more readable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1353/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
added, thanks